container/home/builder/.default_aliases

```diff
@@ -88,12 +88,12 @@ alias ctp-reset="ansible-playbook /srv/inventories/start.yml -e=deploy_mode=rese
 
 ### MISC ###
 
-alias ctp-update-os="ansible-playbook /srv/inventories/start.yml -e=deploy_mode=deploy-role -e=single_role=nova.core.updates -l"
+alias ctp-update-os="ansible-playbook /srv/inventories/start.yml -e=deploy_mode=deploy -e=single_role=nova.core.updates -l"
 
 alias ctp-list="ansible-playbook /srv/inventories/start.yml -e=deploy_mode=deploy --list-hosts -l"
 
 alias ctp-list-vars="ansible-inventory /srv/inventories/start.yml -e=deploy_mode=deploy --vars --host"
 
-alias ctp-rename-vm="ansible-playbook /srv/inventories/start.yml -e=deploy_mode=deploy-role -e pre_role=nova.core.rename_vm -l"
+alias ctp-rename-vm="ansible-playbook /srv/inventories/start.yml -e=deploy_mode=deploy -e pre_role=nova.core.rename_vm -l"
```

Comment:
Also add ` -e role_only=true` after deploy mode. This will skip some of the tasks and make the command faster.

Reply:
But if we also set it when the pre_role, then configure in all the alias that use it no?

Reply:
Yes :)